students/chieu_quach/lesson08/circle_test.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+import pytest
+from circle import Circle, Sphere
+
+def test_default_initial_radius():
+    circle = Circle(2)
+    assert circle.radius == 2
+    
+def test_diameter():
+    circle = Circle(2)
+    assert circle.get_diameter == 4
+
+def test_area():
+    circle = Circle(2)
+    assert round(circle.area(),2) == 12.56
+
+def test_str():
+    c1 = Circle(2)
+    assert str(c1) == '2'
+
+def test_repr():
+    c1 = Circle(2)
+    assert repr(c1) == '2'
+
+def test_add():
+    c  = Circle(2)
+    c2 = Circle(3)
+    assert repr(c + c2) == '5'
+
+def test_mult():
+    c = Circle(2)
+    num = 3
+    assert repr(c * num) == '6'
```

Comment:
your test should assert on class attribute values instead of using `repr()` method.